app-containers/podman/podman-5.0.0_rc1.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module tmpfiles linux-info
+
+DESCRIPTION="A tool for managing OCI containers and pods with Docker-compatible CLI"
+HOMEPAGE="https://github.com/containers/podman/ https://podman.io/"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/podman.git"
+else
+	SRC_URI="https://github.com/containers/podman/archive/v${PV/_rc/-rc}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${P/_rc/-rc}"
+	KEYWORDS="~amd64 ~arm64 ~riscv"
```

Comment:
For future, I'd consider not keywording RCs. It's up to you (as one of its maintainers) but we tend to discourage it, as nowadays, we try to keep ~arch working well rather than experimental things.

We often do stuff like:
`if [[ ${PV} != *rc* ]] ; then ...` to set KEYWORDS.

Reply:
sounds good to me. Thought about doing so cause I also find some instability in basic functionality and had to downgrade to 4.9.2.

May you please do that here in 5.0 and 9999 ebuilds?

Reply:
Updated in c8c28f99d122c186318b8d9719fa253674a22b60. Thanks!

Reply:
Thank you!